### PR TITLE
Post fork cleanups

### PR DESCRIPTION
**Pull request by mattock**
These commits remove some rcrowley-specifisms from Makefile.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,8 +58,8 @@ There's also [French documentation](http://blog.valouille.fr/2014/03/creer-un-de
...
 ## Contribute
 
-Freight is [BSD-licensed](https://github.com/rcrowley/freight/blob/master/LICENSE)
+Freight is [BSD-licensed](https://github.com/freight-team/freight/blob/master/LICENSE)
 
-* Source code: <https://github.com/rcrowley/freight>
-* Issue tracker: <https://github.com/rcrowley/freight/issues>
-* Wiki: <https://github.com/rcrowley/freight/wiki>
+* Source code: <https://github.com/freight-team/freight>
+* Issue tracker: <https://github.com/freight-team/freight/issues>
```
Comment (domcleal) [inline README.md]: This is disabled on the repo right now, you'll need to go to Settings and enable it.

**Comment by domcleal** [comment]
It might be a good idea to add credit to rcowley and perhaps add a line explaining the relationship between the repos.

**Comment by mattock** [comment]
I modified README.md to include the rationale for the fork, plus visibly mention who wrote this (highly useful) application originally. I also enabled the issue tracker, which was disabled. Good catch!

**Comment by domcleal** [comment]
Looks good!

**Comment by mmoll** [comment]
:+1: @mattock could you squash the commits together?

**Comment by mattock** [comment]
@mmoll: ok, sounds reasonable.

**Comment by mattock** [comment]
@mmoll: I squashed the two README.md commits into one, but I'd prefer to keep the two other separate, because they affect the Makefile and do unrelated things. Ok?

**Comment by mattock** [comment]
Once this PR has been merged, I suggest we set the version from 0.3.5 to 0.3.6 in Makefile and produce a new .deb. I can then push the package to our apt repository server on build.openvpn.net, and update README.md accordingly. I can take care of all this myself. Sounds good?

**Comment by domcleal** [comment]
Sounds good!

**Comment by mmoll** [comment]
@mattock sounds good, I'd be interested in getting push acces to this repo also :bow:

**Comment by mattock** [comment]
@mmoll: Done.

**Comment by mmoll** [comment]
merged, thanks @mattock!

**Pull request merged**